src/graphql/desmos_profile_graphql.ts

```diff
@@ -52,3 +52,31 @@ query DesmosProfileLink($address: String) {
   }
 }
 `;
+
+// use this query if using dtag
+export const DesmosProfileDocumentDtag = /* GraphQL */`
```

Comment:
`DesmosProfileDtagDocument`

The suffix Document tells me it's the query's raw string